src/neptune/internal/disk_queue.py

```diff
@@ -177,7 +178,8 @@ def cleanup_if_empty(self) -> None:
         """
         Remove underlying files if queue is empty
         """
-        if self.is_empty():
+        disable_cleanup = os.getenv(NEPTUNE_DISABLE_LOCALFILES_CLEANUP, "false").lower() in ("true", "1", "t")
+        if self.is_empty() and not disable_cleanup:
             self._remove_data()
```

Comment:
`shutil.rmtree(path, ignore_errors=True)` needs to stay

Reply:
Done